video/out/drm_atomic.c

```diff
@@ -209,8 +209,12 @@ struct drm_atomic_context *drm_atomic_create_context(struct mp_log *log, int fd,
                     mp_err(log, "Unable to retrieve type property from plane %d\n", j);
                     goto fail;
                 } else {
-                    if (value == DRM_PLANE_TYPE_CURSOR) // Skip cursor planes
+                    if (value == DRM_PLANE_TYPE_CURSOR) {
```

Comment:
We could probably have a `!=`check here to get into primary / overlay case below and just group the free for all cases :)